dataverse-webapp/src/main/webapp/dataset.xhtml

```diff
@@ -100,7 +100,7 @@
                                                                and DatasetPage.dataset.latestVersion.versionState=='DRAFT'
                                                                and DatasetPage.dataset.owner.released
                                                                and permissionsWrapper.canIssuePublishDatasetCommand(DatasetPage.dataset)
-                                                               and DatasetPage.latestDatasetHasFiles()}">
+                                                               and DatasetPage.latestDatasetWithAnyFilesIncluded()}">
```

Comment:
One last thing: I fear that in this case that syntax could be invalid, it should be `DatasetPage.isLatestDatasetWithAnyFilesIncluded()` or `DatasetPage.latestDatasetWithAnyFilesIncluded`.